diff --git a/node_modules/expo-modules-core/ios/Interfaces/FileSystem/EXFileSystemInterface.h b/node_modules/expo-modules-core/ios/Interfaces/FileSystem/EXFileSystemInterface.h
index fca4649..fa01009 100644
--- a/node_modules/expo-modules-core/ios/Interfaces/FileSystem/EXFileSystemInterface.h
+++ b/node_modules/expo-modules-core/ios/Interfaces/FileSystem/EXFileSystemInterface.h
@@ -9,6 +9,8 @@ typedef NS_OPTIONS(unsigned int, EXFileSystemPermissionFlags) {
 };
 
 // TODO: Maybe get rid of this interface in favor of EXFileSystemManager and private utilities classes
+NS_ASSUME_NONNULL_BEGIN
+
 @protocol EXFileSystemInterface
 
 @property (nonatomic, readonly) NSString *documentDirectory;
@@ -21,3 +23,5 @@ typedef NS_OPTIONS(unsigned int, EXFileSystemPermissionFlags) {
 - (BOOL)ensureDirExistsWithPath:(NSString *)path;
 
 @end
+
+NS_ASSUME_NONNULL_END
diff --git a/node_modules/expo-modules-core/ios/Legacy/ModuleRegistryAdapter/EXModuleRegistryHolderReactModule.h b/node_modules/expo-modules-core/ios/Legacy/ModuleRegistryAdapter/EXModuleRegistryHolderReactModule.h
index 40307db..ad35f3f 100644
--- a/node_modules/expo-modules-core/ios/Legacy/ModuleRegistryAdapter/EXModuleRegistryHolderReactModule.h
+++ b/node_modules/expo-modules-core/ios/Legacy/ModuleRegistryAdapter/EXModuleRegistryHolderReactModule.h
@@ -1,6 +1,6 @@
 // Copyright 2018-present 650 Industries. All rights reserved.
 
-@protocol RCTBridgeModule;
+#import <React/RCTBridgeModule.h>
 
 #import <ExpoModulesCore/EXModuleRegistry.h>
 
